.github/workflows/build-and-release-ctl.yml

```diff
@@ -28,16 +27,42 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.RELEASE_NOTES_PAT }}
           RELEASE_CHANNEL: "stable"
-          Tag: ${{inputs.release-tag}}
+          TAG: ${{inputs.release-tag}}
         with:
           version: 2
           args: release --clean --skip=validate -f .github/.goreleaser.yml
+
+        # ---------------- Cosign signing step----------------
+      - name: Install Cosign
+        uses: sigstore/cosign-installer@v3.5.0
+
```

Comment:
It looks like recent version is 3.9.2 any reason to pin it to a previous version